specification/cognitiveservices/AnomalyDetector/tspconfig.yaml

```diff
@@ -17,7 +17,7 @@ options:
     examples-directory: ./examples
     omit-unreachable-types: true
     output-file: openapi.json
-    emitter-output-dir: "{project-root}/../data-plane/AnomalyDetector/stable"
+    emitter-output-dir: "{project-root}/../data-plane/AnomalyDetector/stable/v1.1"
```

Comment:
Should `output-file` and `emitter-output-dir` be changed to align with the sample?  Something like this:

```
    emitter-output-dir: "{project-root}/.."
    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/openapi.json"
```

In particular, I thought `output-file` is supposed to use a template expression after the recent breaking change:

https://azure.github.io/typespec-azure/docs/release-notes/release-2023-04-11#azure-toolstypespec-autorest-config-interpolation

Reply:
Also, why is the version `v1.1` instead of a date-based version like `2022-11-01-preview`?

Reply:
This was just to compare with their original swagger.  If we would like to move it, we cna use the more conventional config setting

Reply:
I wonder if we can make the config more aligned to the sample, but still use the existing swagger folder structure?  I'll try it locally.

Reply:
This config works for me locally, and is closer to the sample:

```
azure-resource-provider-folder: "./data-plane"
emitter-output-dir: "{project-root}/.."
examples-directory: ./examples
output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/openapi.json"
```

Reply:
Oh, you're totally right.  I missed that they hav 'v1.1' as the actual version

Reply:
> Oh, you're totally right. I missed that they hav 'v1.1' as the actual version

I'm not sure if it's a bug or by design to have `v` in the version, but yeah that's why it works.